rust/agents/validator/src/validator.rs

```diff
@@ -286,6 +293,39 @@ impl Validator {
         }
     }
 
+    async fn announce_to_avs(&self) -> Result<()> {
```

Comment:
The tx to submit this onchain isn't done by the agent right?

Wondering if we should make this function and `write_operator_registration` more consistent in some way - as someone who isn't familiar with eigenlayer, it's not immediately clear that both are related / are related to EL

Reply:
yes, that makes sense. Fixed